test/support/single.spec.js

```diff
@@ -1,6 +1,6 @@
 require('./../utils/starter.spec.js').runTests({
     description: 'In print utilities, ',
```

Comment:
So the plan here is:
- mv `single.spec.js` to `single.spec.js.sample`
- add `test/support/*.spec.js` to the `.gitignore`